mungegithub/mungers/sync/issue-sync.go

```diff
@@ -50,7 +45,7 @@ type Priority int
 
```

Comment:
change the type of Priority to string.  And then initialize it with its verbose name rather than the number 2.